sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -191,7 +192,9 @@ std::unique_ptr<RawResponse> CurlTransport::Send(Context const& context, Request
         "Error while sending request. " + std::string(curl_easy_strerror(performing)));
```

Comment:
@vhvb1989 - let's review the places where we throw exceptions and log and `Error` level message following:
https://azure.github.io/azure-sdk/general_implementation.html#logging